lectures/lln_clt.md

```diff
@@ -438,7 +438,7 @@ Here $\stackrel { d } {\to} N(0, \sigma^2)$ indicates [convergence in distributi
 
 The striking implication of the CLT is that for **any** distribution with
 finite [second moment](https://en.wikipedia.org/wiki/Moment_(mathematics)), the simple operation of adding independent
-copies **always** leads to a Gaussian curve.
+copies **always** leads to a Gaussian(Normal) curve.
```

Comment:
```suggestion
copies **always** leads to a Gaussian (Normal) curve.
```

This is a really minor point but @jstac re: style I think there should be space between Gaussian and (Normal). Do you agree?

Reply:
Yes please, thanks @mmcky and @SylviaZhaooo